<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNSampleDataWidget</class>
 <widget class="QWidget" name="HNSampleDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="HNMPTableWidget" name="widgetSampleTable" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>674</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLeft">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRight">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrint">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>print</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExport">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>310</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>310</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HNMPTableWidget</class>
   <extends>QWidget</extends>
   <header>hnmptablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
